.gitignore

```diff
@@ -2,6 +2,10 @@
 .idea/
 .env
 
+# package manager
+package-lock.json
+yarn.lock
```

Comment:
We don't want to ignore these lock files

Reply:
oh, let me fix it again